Sources/TiltUp/Screens/Camera/Views/CameraOverlayView.swift

```diff
@@ -245,7 +245,7 @@ final class CameraOverlayView: UIView {
         return outerStackView
     }()
 
-    private lazy var previewImageView: UIImageView = {
+    public lazy var previewImageView: UIImageView = {
```

Comment:
is this change needed?

Reply:
Yes, its now used to set the previewView frame in the CameraController